src/wabt.cpp

```diff
@@ -61,6 +61,10 @@ class WabtEthereumInterface : public EthereumInterface {
   size_t memorySize() const override { return m_wasmMemory->data.size(); }
   void memorySet(size_t offset, uint8_t value) override { m_wasmMemory->data[offset] = static_cast<char>(value); }
   uint8_t memoryGet(size_t offset) override { return static_cast<uint8_t>(m_wasmMemory->data[offset]); }
+  uint8_t* memoryPointer(size_t offset, size_t length) override {
+    ensureCondition(memorySize() >= (offset + length), InvalidMemoryAccess, "Memory is shorter than requested segment");
```

Comment:
I'm not sure the "shorter" is the right word.

Reply:
Can you change it to "Memory (size) is smaller ..."?

Reply:
Did find this instance in Boost where they the same sentence :wink::
>   * @param n The number of bytes to return in the buffer sequence. If the
>   * underlying memory is shorter, the buffer sequence represents as many bytes
>   * as are available.

I can change it if it is important.